public/plugins/grafana-testdata-datasource/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@grafana-plugins/testdata",
+  "version": "1.0.0",
```

Comment:
Is this correct? At build time it is used to populate the version field in `package.json`.

Reply:
you mean `plugin.json`, right? It's the default version when you create a plugin. Since it has never been published I think it's a good "first" version.

Reply:
Sorry, yes I mean `plugin.json`.